frontend/system/src/Admin/EditEntity/helpers.test.ts

```diff
@@ -9,26 +9,33 @@ import {
   isValidDrugInput,
   isValidVaccineInput,
 } from './helpers';
-import { ConsumableInput, DrugInput, VaccineInput } from './types';
+import {
+  ConsumableInput,
+  DrugInput,
+  EntityDetails,
+  VaccineInput,
+} from './types';
 
 describe('getAllEntityCodes', () => {
   it('returns empty array when entity is not defined', () => {
     expect(getAllEntityCodes(undefined)).toMatchObject([]);
   });
   it('returns array of codes with empty strings filtered out', () => {
-    const entityDetails = {
+    const entityDetails: EntityDetails = {
       code: '7c8c2b5b',
       name: 'Acetic Acid',
       type: 'drug',
       alternativeNames: [],
       properties: [],
+      barcodes: [],
```

Comment:
these type issues should have been caught earlier... I think this is another instance of where not running TS compilation before tests, nor ensuring that the tests themselves compile, is a little unfortunate